execution-engine/engine-tests/benches/transfer_bench.rs

```diff
@@ -0,0 +1,147 @@
+    EngineConfig::new().set_use_payment_code(true)
+}
+
+fn bootstrap(accounts: &[PublicKey]) -> (WasmTestResult<LmdbGlobalState>, TempDir) {
+    println!("Creating {} accounts...", accounts.len());
+    let accounts_bytes: Vec<Vec<u8>> = accounts
+        .iter()
+        .map(|public_key| public_key.value().to_vec())
+        .collect();
+    let amount = U512::from(1);
+
+    let data_dir = TempDir::new().expect("should create temp dir");
+    let result = LmdbWasmTestBuilder::new_with_config(&data_dir.path(), engine_with_payments())
+        .run_genesis(GENESIS_ADDR, HashMap::new())
+        .exec_with_args(
+            GENESIS_ADDR,
+            STANDARD_PAYMENT_CONTRACT,
+            (U512::from(MAX_PAYMENT),),
+            "create_accounts.wasm",
+            (accounts_bytes, amount), //args
+            DEFAULT_BLOCK_TIME,       // blocktime
+            [1; 32],                  // deploy_hash
+        )
+        .expect_success()
+        .commit()
+        .finish();
+
+    // println!("Bootstrap finished in {}ms", now.elapsed().as_millis());
+    (result, data_dir)
+}
+
+fn visit_dirs(dir: &Path, cb: &dyn Fn(&DirEntry)) -> io::Result<()> {
+    if dir.is_dir() {
+        for entry in fs::read_dir(dir)? {
+            let entry = entry?;
+            let path = entry.path();
+            if path.is_dir() {
+                visit_dirs(&path, cb)?;
+            } else {
+                cb(&entry);
+            }
+        }
+    }
+    Ok(())
+}
+
+fn clone_directory(source: &Path) -> TempDir {
+    let dest = TempDir::new().expect("should create temp dir");
+    visit_dirs(&source, &|dir_entry: &DirEntry| {
+        // Source
+        let source_path = dir_entry.path();
+        // Destination
+        let mut dest_path = dest.path().to_path_buf();
+        dest_path.push(dir_entry.path().file_name().unwrap());
+        assert_ne!(source_path, dest_path);
+        fs::copy(source_path, dest_path).unwrap();
+    })
+    .expect("should visit");
+    dest
+}
+
+fn exec_send_to_account(builder: &mut LmdbWasmTestBuilder, accounts: &[PublicKey]) {
+    let amount: U512 = 1u64.into();
+    // To see raw numbers take current time
+    for (i, account) in accounts.iter().enumerate() {
+        builder
+            .exec_with_args(
+                GENESIS_ADDR,
+                STANDARD_PAYMENT_CONTRACT,
+                (U512::from(MAX_PAYMENT),),
+                "send_to_account.wasm",
+                (*account, amount), //args
+                DEFAULT_BLOCK_TIME, // blocktime
+                [2 + i as u8; 32],  // deploy_hash
+            )
+            .expect_success()
+            .commit();
+    }
+    // NOTE: To see raw numbers here calculate tps as following:
+    // accounts.len() / now.elapsed().as_millis() * 1000;
+}
+
+#[macro_use]
+extern crate criterion;
+extern crate contract_ffi;
+extern crate engine_core;
+extern crate engine_shared;
+extern crate engine_storage;
+
+use criterion::black_box;
```

Comment:
```suggestion
use criterion::{self, Criterion, Throughput};
```
(and remove the following 2 lines)

(see below for `black_box` use site)